content/en/_index.md

```diff
@@ -16,69 +15,85 @@ cascade:
     path: "/**"
 ---
 
-This page includes links to other Cobalt literature, including our
-[Getting Started](/getting-started/) guide.
+<div style="padding-right: 2.0625rem;">
```

Comment:
@healqq Can you please check the latest change? I added a `div` to change the padding on the right. It's a hack solution, but I don't know a better way.

The default padding comes from the theme we're using:
- Left: 48px
- Right: 15px

I calculated that 33px is 2.0625 rem.

Reply:
that works, maybe we can adjust spacing on <main /> in follow up task to not need this anymore